source/cfp.rst

```diff
@@ -0,0 +1,107 @@
+Call For Proposals
+==================
+
+We are planning to use Google Forms for handling CFP proposals.
+
+Daniele has set up a form with the following questions:
+
+About you
+---------
+
+We'd like to know a little bit about you for our files. If you submit more than
+one proposal, you don't need to send all of these details again.
```

Comment:
It does say _all of these details again_, not _any of these details again_. And we need to make sure we get it at least once.